website/docs/enterprise/getting-started/install-enterprise-cli.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Install Weave GitOps Enterprise via CLI
+hide_title: true
+toc_max_heading_level: 4
+---
+
+import TierLabel from "../../_components/TierLabel";
+
+# Install Weave GitOps Enterprise via CLI
+
+## Prerequisites
+
+#### Git Repository
+
+To be used as the repository of the WeaveGitOps Enterprise repo.
+
+#### Flux Installed Locally (binary)
+
+To be used in reconciling sources, Kustomizations, HelmReleases
+
+#### Kubectl binary
+
+To be used in port-forwarding
+
+#### A Kubernetes Cluster
+
+To install weave gitops enterprise. We also require a kubeconfig with admin permissions to create resources.
+
+#### Entitlments File
+
+To get the entitlement secret please contact *sales@weave.works*, then apply it on your management cluster with the name `weave-gitops-enterprise-credentials` under `flux-system` namespace. 
+
+#### Flux bootstrapped via SSH
+
+To boostrap Weave Gitops we require flux to be bootstrapped using the configuration repo specified earlier via SSH [here](https://fluxcd.io/flux/cmd/flux_bootstrap_git/)
+
+#### SSH private key for Git Repo 
+
+Bootstrap command writes manifests to the git repo via ssh.
+
+#### [Phase 1] Flux on Management Cluster 
+
+On the management cluster, Flux (>v1.x.x) needs to be installed on the management cluster. There are different ways to bootstrap flux on your Kubernetes clusters, please follow the instructions found on flux bootstrap documentation [here](https://fluxcd.io/flux/installation/bootstrap/)
+
+#### [Phase 2] Flux on Management Cluster (Optional)
+
+If your cluster doesn’t have flux installed on the management cluster, `gitops-ee bootstrap` offer a way to bootstrap flux using the git generic approach. If you need more tailored or advanced configurations for the installed flux binary installed on your cluster, please follow the instructions found on flux bootstrap documentation [here](https://fluxcd.io/flux/installation/bootstrap/).  
+
+#### `gitops-ee` CLI v.?.?
+
+Weave GitOps Enterprise Bootstrap functionality is available on Weave GitOps Enterprise CLI starting from version x.x.x. Use the following command to install the latest `gitops-ee` CLI.
+`brew install weaveworks/tap/gitops-ee`
+
+## Installing Weave GitOps Enterprise via CLI
+
+Please use the following command to start the installation wizard of Weave GitOps Enterprise.
+
+```bash
+gitops-ee bootstrap
+```
+
+The bootstrap wizard will take you step-by-step into configuring Weave GitOps Enterprise. To understand more about the CLI configurations experience, check the below sections [here](#cli-configurations). 
```

Comment:
do you think that we should borrow some ideas from [flux bootstrap git](https://fluxcd.io/flux/cmd/flux_bootstrap_git/)

for example, i could think that at least the main usage modes would be good to document: 
- a user that wants non-interactive execution (for example cause using it via automation)
- a user that wants an interactive execution

Reply:
we should also consider also like the `using-defaults` mode